pcd_map_localization:
  # Use for global map down sample
  localizingGlobalMapLeafSize: 1.0
  localizingDeskewedLeafSize: 2.0
  globalMapDsSearchRadius: 100
  ndtResolution: 1.2
  ndtMaxIterations: 1000
  ndtRotationEpsilon: 0.00001
  # Read global map in given path or subscrible from rosbag Topic.
  toRosBag: true                               # if true save message to rosbag
  savedPCDDirectory: "/Datasets/PCDMap/lio_sam_walking/"        # start at your home folder: /home/${user}/, starts and ends with "/".
  savedCornerMapDirectory: "corner/" # savePCDDirectory + saveCornerMapDirectory, start ""
  savedSurfMapDirectory: "surf/" # savePCDDirectory + saveSurfrMapDirectory, start ""
  savedDeskewedPointcloudDirectory: "input_data/" # savePCDDirectory + saveSurfrMapDirectory, start ""
  
  # If loadPCD is false, subscrible these topics to load priority map.
  # NOTE: This .pcd format map can generate by lio_sam or others methods.
  #       Localization based on a priority map means we daopt global map as absolute reference.
  #       Ground truth (GT) of trajectory and transform also generate by lio_sam algorithm, also support others.
  #       About the detailed format of these topics message please click https://github.com/mosthuiyan/lio_sam_map_localization
  globalCornerSplitsTopic: "global_map/corner_splits"
  globalSurfSplitsTopic: "global_map/surf_splits"
  globalCornerMapTopic: "global_map/corner"
  globalSurfMapTopic: "global_map/surf"
  globalMapTopic: "global_map/global"
  gtTrajectoryTopic: "global_map/gt_trajectory"
  gtTransformationsTopic: "global_map/gt_transformations"
  deskewedPointcloudTopic: "input/deskewed_pointcloud"

  # Coordinate name in ros
  worldCoordinateName: "world"
  robotCoordinateName: "robot"
  lidarCoordinateName: "lidar"
  odometeryCoordinateName: "odometery"

  # NOTE: this initial pose is provided by God perspective